app/controllers/users_controller.rb

```diff
@@ -66,6 +67,46 @@ def index
              end
   end
 
+  def all_admins
```

Comment:
I think all these should go into `SettingsController` or a more specific new controller, like `UserSettingsController`.

`UserController` is already pretty crowded, and these are not general-purpose user actions.